troubleshooting-test/apps/vehicles/src/main/java/com/redhat/vehicles/events/VehicleMoved.java

```diff
@@ -0,0 +1,30 @@
+package com.redhat.vehicles.events;
+
+/**
+ * This event occurs when a vehicle moves
+ */
+public class VehicleMoved {
+    public int vehicleId;
```

Comment:
I think it would be better if we encapsulated these fields. It is still an unwritten rule for Java apps I assume.

Reply:
I'm adding this as a TODO comment